app/assets/javascripts/initializers/initializeCommentDropdown.js

```diff
@@ -65,7 +67,9 @@ function initializeCommentDropdown() {
       const clipboardCopyElement = document.getElementsByTagName(
         'clipboard-copy',
       )[0];
-      clipboardCopyElement.removeEventListener('click', iOSCopyText);
+      if (document.body.contains(clipboardCopyElement)) {
+        clipboardCopyElement.removeEventListener('click', iOSCopyText);
+      }
```

Comment:
This is hiding a larger, problem. On lines 64 and 99 of the master copy, the if is not calling the `isIOSDevice` function, just checking if the `isIOSDevice` variable is not undefined, which being always true, ends up in that if.

Reply:
also, why not just checking if `clipboardCopyElement` exists?

Reply:
I did see that `isIOSDevice` was functioning that way since the method was getting called everytime. I will switch to a proper check for the same.

Should I use `!!document.getElementById('clipbaordCopyElement')`? Thought it would be better to check the Visible dom for it.

Reply:
why not just `if (element)` ? `document.getElementById('clipbaordCopyElement')` returns null if the element is not present, and null in javascript never is "falsey"

Reply:
I have never tried that approach, hehe, time to research a bit on that behavior.